src/app.jsx

```diff
@@ -439,22 +440,14 @@ class Application extends React.Component {
 
         let startService = "";
         if (!this.state.systemServiceAvailable && permission.allowed) {
-            startService = <div className="alert alert-info dialog-info">
-                <div className="info-message">
-                    <span className="fa fa-exclamation-triangle" />
-                    <span>{_("System Podman service is also available")}</span>
-                </div>
-                <button onClick={this.startService}>{_("Start")}</button>
-            </div>;
+            startService = <Alert isInline variant='default'
```

Comment:
I think this should not be inlined in the search panel on top, but it should be just underneath it.